Demo1.js

```diff
@@ -27,6 +27,15 @@ export default React.createClass({
     };
   },
 
+  handleTouchStart: function(key, pressLocation, event) {
+    this.handleMouseDown(key, pressLocation, event.touches[0]);
```

Comment:
Hey what's `touches[0]` again? Would it be clearer if we used destructuring?
`{touches: [something]}`